src/org/omegat/gui/main/MainWindowUI.java

```diff
@@ -279,21 +279,18 @@ public static void initializeScreenLayout(MainWindow mainWindow) {
     }
 
     /**
-     * Assume screen size is 800x600 if width less than 900, and 1024x768 if
-     * larger. Assume task bar at bottom of screen. If screen size saved,
-     * recover that and use instead (18may04).
+     * (20nov22) We don't  worry  about default
+     * size because if the screen is too small for the setting, OmegaT
+     * will just use the whole window size. So I guess we just have to
+     * consider a maximum useable width for big screens.
+     * Let's assume the "standard" is 1920x1080 for "pro" laptops
      */
+
     static Rectangle getDefaultBounds() {
         // size info missing - put window in default position
         GraphicsEnvironment env = GraphicsEnvironment.getLocalGraphicsEnvironment();
         Rectangle scrSize = env.getMaximumWindowBounds();
-        if (scrSize.width < 900) {
-            // assume 800x600
-            return new Rectangle(0, 0, 580, 536);
-        } else {
-            // assume 1024x768 or larger
-            return new Rectangle(0, 0, 690, 700);
-        }
+        return new Rectangle(210, 0, 1500, 1080);
```

Comment:
What's the point of assigning the `scrSize` variable since you're not using it?

Did you mean to assign `Rectangle(210, 0, 1500, 1080)` or the result of `env.getMaximumWindowBounds()` if the latter is smaller?

Reply:
I just left the code there. Obviously it can be removed.

Reply:
Could you make a proposal in the GitHub interface as Hiroshi does ?

Reply:
Did I understand your intent correctly?

If so, I'll try to propose appropriate code later this evening.

Reply:
There is no need to test the screen size, since OmegaT will use the whole screen if the size is too small.

The point is to set a size that is not too small for big screens. Smaller screens don't have this problem.

Reply:
OK.

I've just proposed an alternative that eliminates the check and removes redundant code.
Please take a look.